src/main/resources/job-scheduler-config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "projectId": "client-side-events",
+  "configuration":
+  {
+    "query":
+    {
+      "writeDisposition": "WRITE_TRUNCATE",
+      "destinationTable":
+      {
+        "projectId": "client-side-events",
+        "datasetId": "Aggregate_Tables"
+      }
+    }
+  },
+
+  "every4Hours": [
```

Comment:
@tejohnso that wouldn't be a lot of work to implement, but I feel it might make configuring queries more arduous than it needs to be. Instead of putting a query under the 4-hour interval, you'd have to add `[0, 4, 8, 12, 16, 20]`